src/gui/shareusergroupwidget.h

```diff
@@ -19,13 +19,15 @@
 #include "sharemanager.h"
 #include "sharepermissions.h"
 #include "sharee.h"
+#include "profilepagewidget.h"
 #include "QProgressIndicator.h"
 #include <QDialog>
 #include <QWidget>
 #include <QSharedPointer>
 #include <QList>
 #include <QVector>
 #include <QTimer>
+#include <qpushbutton.h>
```

Comment:
@FlexW Why not `#include <QPushButton>` as in 99% of other sources?

Reply:
See my other comment :)